README.md

```diff
@@ -63,7 +63,87 @@ Body:
     }
 ]
 ```
-
+### List one specific food by id in the database
+Request:
+```
+GET /api/v1/foods/:id
+Accept: application/json
+```
+Example response:
+```
+Status: 200
+Content-Type: application/json
+Body:
+{
+    "id": 1,
+    "name": "Banana",
+    "calories": 150
+}
+```
+### Create a new food
+Request:
+```
+POST /api/v1/foods
+Accept: application/json
+```
+Both name and calories are required fields.
+Pass in the following parameters in the body of the request:
+```
+{
+    "food": {
+        "name": "Name of food here",
+        "calories": "Calories here"
+      }
+}
+```
+Example response:
+```
+Status: 200
+Content-Type: application/json
+Body:
+{
+    "id": 1,
+    "name": "Banana",
+    "calories": 150
+}
+```
+Failed post response (did not include name or calorie):
+```
+Status: 400
+Body:
+{ "error": '"name" is a required field.' }
+```
+### Update an existing food by id in the database
+Request:
+```
+PATCH /api/v1/foods/:id
+Accept: application/json
+```
+Pass in the following parameters in the body of the request:
+```
+{
+    "food":
```

Comment:
Added issue for this -- not actually reading in data this way right now. Can resolve on a different branch.

Reply:
Resolved in this branch